build.gradle

```diff
@@ -390,35 +390,9 @@ task bundle(type: (isWindows()) ? Zip : Tar) {
     }
     from(project.rootDir) {
         include 'RELEASE_NOTES.md'
-        include 'samples/**'
```

Comment:
Btw, this breaks our TeamCity tests, because samples are being looked for in the bundle.
cc @ilmat192

Reply:
Ok, thank you for the clarification.